python/servo/visual_studio.py

```diff
@@ -30,7 +31,7 @@ class VisualStudioInstallation:
     installation_path: str
     vc_install_path: str
 
-    def __lt__(self, other) -> bool:
+    def __lt__(self, other: "VisualStudioInstallation") -> bool:
```

Comment:
Why is the type name in a string?


```suggestion
    def __lt__(self, other: "VisualStudioInstallation") -> bool:
```